test/warn-decide-predicate-unknown/lemmas-no-smt-spec.k.out.golden

```diff
@@ -0,0 +1,3 @@
+<k>
+  doneLemma ( bool2Word ( B ) ==Int 1 ) ~> _DotVar1 ~> .
```

Comment:
I don't understand this test. What would have been the output without your changes?

Reply:
yeah this is why i asked about the integration test above. the final result is the same, but this test now throws some warning for the simplification rules before throwing an error and exiting. We could maybe change the test script to just check it produces a warning?